rbac.yaml

```diff
@@ -27,6 +27,9 @@ rules:
 - apiGroups: ["policy"]
   resources: ["poddisruptionbudgets"]
   verbs: ["create", "delete", "get"]
+- apiGroups: [""]
+  resources: ["configmaps", "persistentvolumeclaims"]
+  verbs: ["create", "delete", "get" , "update"]
```

Comment:
@danni-m for PVC - we need create, update, get (harmless), but not sure about delete?